.github/workflows/publish-test-results.yml

```diff
@@ -78,9 +78,9 @@ jobs:
         if: ${{ github.event.workflow_run.event == 'pull_request' }}
         with:
           header: Test Results
-          recreate: true
+          recreate: false
```

Comment:
@oren-zohar, according to [plugin's documentation](https://github.com/marocchino/sticky-pull-request-comment) **recreate** parameter is optional, and indicates if previous comment should be removed before creating a new comment. Only true is allowed. Just skip this option when you don't need it. Thus if not used can be removed.
But if I understand correctly, if option is **false**, each execution will create a new comment with the results link.